packages/monaco-graphql/package.json

```diff
@@ -45,7 +65,7 @@
   },
   "peerDependencies": {
     "graphql": "^15.5.0 || ^16.0.0",
-    "monaco-editor": ">= 0.20.0 < 1",
+    "monaco-editor": ">= 0.39.0 < 1",
```

Comment:
update as per Rikki suggestion for next major

Reply:
is there a reason we need `0.39.0` specifically? is there a more forgiving range we can support? we should document what we use from the `0.39.0` release that deprecates support for `0.38.0` etc

Reply:
No reason, which version you propose to bump?

Reply:
@acao I was rethinking this, since in `monaco-editor` every `minor` version is `major`, according to semver policy (since major is `0`) let's bump it as highest as possible (`0.40.0` in this case)